Makefile

```diff
@@ -154,11 +165,23 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 		&& $(KUSTOMIZE) edit set namespace ${OPERATOR_NS}
 	$(KUSTOMIZE) build config/overlays/make-deploy | kubectl apply -f -
 
+.PHONY: deploy-kind
+deploy-kind:
+	cd config/overlays/kind-tests \
+		&& kustomize edit set image controller=${IMG} \
+		&& kustomize edit set namespace ${OPERATOR_NS}
+	kustomize build config/overlays/kind-tests | kubectl apply -f -
```

Comment:
`kustomize` -> ${KUSTOMIZE}

Reply:
the reason why I opted not to use this is because kind clusters are being utilized in gh actions, where Kustomize is already present and  ${KUSTOMIZE} is set via the make command `make kustomize` which will download kustomize, we could add a "if it exists don't download" but I feel like that's out of scope for this pr.